packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -77,7 +77,7 @@ const NuovaDelega = () => {
   const isMobile = useIsMobile();
   const dispatch = useAppDispatch();
   const { created } = useAppSelector((state: RootState) => state.newDelegationState);
-
+  const { entities } = useAppSelector((state: RootState) => state.newDelegationState);
```

Comment:
@AndriBoca riga 79 e 80 possono essere scritte in un'unica riga  const { entities, created } = useAppSelector((state: RootState) => state.newDelegationState);